packages/scfg/scfg.0.1/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis:
+  "OCaml library and executable to work with the scfg configuration file format"
+description: """
+scfg is an OCaml library and executable to work with the scfg configuration file format.
+It provides a parser, a pretty printer and a module to perform queries.
+"""
+maintainer: ["Léo Andrès <contact@ndrs.fr>"]
+authors: ["Léo Andrès <contact@ndrs.fr>"]
+license: "ISC"
+tags: ["scfg" "configuration" "format" "simple" "config" "parser" "printer"]
+homepage: "https://git.zapashcanon.fr/zapashcanon/scfg"
+doc: "https://doc.zapashcanon.fr/scfg"
+bug-reports: "https://git.zapashcanon.fr/zapashcanon/scfg/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.13"}
+  "menhir"
```

Comment:
A lower-bound constraint is missing here:
```
# (cd _build/default && /home/opam/.opam/4.13/bin/menhir src/menhir_parser.mly --base src/menhir_parser --infer-write-query src/menhir_parser__mock.ml.mock)
# /home/opam/.opam/4.13/bin/menhir: unknown option '--infer-write-query'.
# Usage: /home/opam/.opam/4.13/bin/menhir <options> <filenames>
#   --base <basename>           Specifies a base name for the output file(s)
#   --canonical                 Construct a canonical Knuth LR(1) automaton
#   --comment                   Include comments in the generated code
#   --compare-errors <filename> (used twice) Compare two .messages files.
#   --compile-errors <filename> Compile a .messages file to OCaml code.
#   --coq                       Generate a formally verified parser, in Coq
#   --coq-no-complete           Do not generate a proof of completeness
#   --coq-no-actions            Ignore semantic actions in the Coq output
#   --depend                    Invoke ocamldep and display dependencies
#   --dump                      Describe the automaton in <basename>.automaton
#   --echo-errors <filename>    Echo the sentences in a .messages file
#   --error-recovery            (no longer supported)
#   --explain                   Explain conflicts in <basename>.conflicts
#   --external-tokens <module>  Import token type definition from <module>
#   --fixed-exception           Declares Error = Parsing.Parse_error
#   --follow-construction       (undocumented)
#   --graph                     Write grammar's dependency graph to <basename>.dot
#   --infer                     Invoke ocamlc for ahead of time type inference
#   --inspection                Generate the inspection API (requires --table)
#   --interpret                 Interpret the sentences provided on stdin
#   --interpret-show-cst        Show a concrete syntax tree upon acceptance
#   --interpret-error           Interpret one sentence that should end in an error
#   --lalr                      Construct an LALR(1) automaton
#   --list-errors               Produce a list of erroneous inputs
#   --log-automaton <level>     Log information about the automaton
#   --log-code <level>          Log information about the generated code
#   --log-grammar <level>       Log information about the grammar
#   --no-code-inlining          (undocumented)
#   --no-inline                 Ignore the %inline keyword.
#   --no-pager                  (undocumented)
#   --no-prefix                 (undocumented)
#   --no-stdlib                 Do not load the standard library
#   --ocamlc <command>          Specifies how ocamlc should be invoked
#   --ocamldep <command>        Specifies how ocamldep should be invoked
#   --only-preprocess           Print grammar and exit
#   --only-preprocess-u         Print grammar with unit actions and exit
#   --only-preprocess-uu        Print grammar with unit actions & tokens and exit
#   --only-tokens               Generate token type definition only, no code
#   --raw-depend                Invoke ocamldep and echo its raw output
#   --stdlib <directory>        Specify where the standard library lies
#   --strict                    Warnings about the grammar are errors
#   --suggest-comp-flags        Suggest compilation flags for ocaml{c,opt}
#   --suggest-link-flags-byte   Suggest link flags for ocamlc
#   --suggest-link-flags-opt    Suggest link flags for ocamlopt
#   --suggest-menhirLib         Suggest where is MenhirLib
#   --suggest-ocamlfind         Show whether Menhir was installed using ocamlfind
#   --table                     Use the table-based back-end
#   --timings                   Display internal timings
#   --trace                     Include tracing instructions in the generated code
#   --unused-token <token>      Do not warn that <token> is unused
#   --unused-tokens             Do not warn about any unused token
#   --update-errors <filename>  Update auto-comments in a .messages file
#   --version                   Show version number and exit
#   -b <basename>               Synonymous with --base <basename>
#   -lg                         Synonymous with --log-grammar
#   -la                         Synonymous with --log-automaton
#   -lc                         Synonymous with --log-code
#   -t                          Synonymous with --table
#   -v                          Synonymous with --dump --explain
#   -help                       Display this list of options
#   --help                      Display this list of options
```

Reply:
@kit-ty-kate fixed